### PR TITLE
[docs] Add guide about installing Deckhouse in kind

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Add a guide about installing Deckhouse in kind.

## Changelog entries
```changes
section: docs
type: feature
summary: A guide about installing Deckhouse in kind.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

File: docs/site/LOCAL_DEV.md [hunk 1]
```diff
@@ -2,32 +2,37 @@
...
-   export BASE_JEKYLL=jekyll/jekyll:3.8@sha256:9521c8aae4739fcbc7137ead19f91841b833d671542f13e91ca40280e88d6e34 
-   werf compose up --follow --docker-compose-command-options='-d'
-   ```
-4. Open http://localhost
-
-5. Don't forget to stop documentation and site containers by running:
-   ```shell
-   werf compose down
-   ``` 
+Free 80 port to bind
```
Comment (Zhbert) [inline docs/site/LOCAL_DEV.md:5]:
```suggestion
Free 80 port to bind.
```

File: docs/site/LOCAL_DEV.md [hunk 2]
```diff
@@ -2,32 +2,37 @@
...
+
+Open console and start documentation container
```
Comment (Zhbert) [inline docs/site/LOCAL_DEV.md:7]:
```suggestion
Open console and start documentation container:
```

File: docs/site/LOCAL_DEV.md [hunk 3]
```diff
@@ -2,32 +2,37 @@
...
+cd docs/documentation
+docker network create deckhouse
+export BASE_NGINX_ALPINE=nginx:1.15.12-alpine@sha256:57a226fb6ab6823027c0704a9346a890ffb0cacde06bc19bbc234c8720673555
+export BASE_ALPINE=alpine:3.12.1@sha256:c0e9560cda118f9ec63ddefb4a173a2b2a0347082d7dff7dc14272e7841a5b5a
+export BASE_GOLANG_16_ALPINE=golang:1.16.3-alpine3.12@sha256:371dc6bf7e0c7ce112a29341b000c40d840aef1dbb4fdcb3ae5c0597e28f3061
+export BASE_JEKYLL=jekyll/jekyll:3.8@sha256:9521c8aae4739fcbc7137ead19f91841b833d671542f13e91ca40280e88d6e34 
+werf compose up --follow --docker-compose-command-options='-d'
+```
+
+Open separate console and start site container
```
Comment (Zhbert) [inline docs/site/LOCAL_DEV.md:19]:
```suggestion
Open separate console and start site container:
```

File: docs/site/LOCAL_DEV.md [hunk 4]
```diff
@@ -2,32 +2,37 @@
...
+source $(trdl use werf 1.2 ea)
+cd docs/site
+export BASE_NGINX_ALPINE=nginx:1.15.12-alpine@sha256:57a226fb6ab6823027c0704a9346a890ffb0cacde06bc19bbc234c8720673555
+export BASE_ALPINE=alpine:3.12.1@sha256:c0e9560cda118f9ec63ddefb4a173a2b2a0347082d7dff7dc14272e7841a5b5a
+export BASE_GOLANG_16_ALPINE=golang:1.16.3-alpine3.12@sha256:371dc6bf7e0c7ce112a29341b000c40d840aef1dbb4fdcb3ae5c0597e28f3061
+export BASE_JEKYLL=jekyll/jekyll:3.8@sha256:9521c8aae4739fcbc7137ead19f91841b833d671542f13e91ca40280e88d6e34 
+werf compose up --follow --docker-compose-command-options='-d'
+```
+
+Open http://localhost
```
Comment (Zhbert) [inline docs/site/LOCAL_DEV.md:30]:
```suggestion
Open http://localhost.
```

File: docs/site/_includes/getting_started/global/partials/INSTALL_OTHER_RU.liquid [hunk 5]
```diff
@@ -6,31 +6,31 @@
...
 {% endif %}{% if page.platform_type == "cloud" %} -v "$PWD/resources.yml:/resources.yml" -v "$PWD/dhctl-tmp:/tmp/dhctl" {% endif %} registry.deckhouse.io/deckhouse/ce/install:stable bash
 ```
 {% endsnippetcut %}
 {% endif %}
 
-{%- if page.platform_type == "existing" %}
+{%- if page.platform_type == "existing" or page.platform_code == "kind" %}
 Примечание:
-- В kubeconfig необходимо смонтировать kubeconfig с доступом к Kubernetes API.
+- В файл `/kubeconfig` контейнера необходимо смонтировать файл конфигурации kubectl с доступом к Kubernetes API. В руководстве предполагается, что он находится в файле `.kube/config` домашней папки пользователя.
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/global/partials/INSTALL_OTHER_RU.liquid:26]:
```suggestion
- В файл `/kubeconfig` контейнера необходимо смонтировать файл конфигурации kubectl с доступом к Kubernetes API. В руководстве предполагается, что он находится в файле `.kube/config` домашнего каталога пользователя.
```

File: docs/site/_includes/getting_started/kind/STEP_FINISH.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+<div markdown="1">
+## Congratulations, your Deckhouse platform is up and running!
+
+Now that you have installed and properly configured Deckhouse, let's look at what you can do with it.
+
+Grafana is available at [grafana-127-0-0-1.nip.io](http://grafana-127-0-0-1.nip.io).
+
+Access to Grafana is restricted via the basic authentication mechanism (additional authentication options are provided in the [user-auth](/{{ page.lang }}/documentation/v1/modules/150-user-authn/) module:
+- Login — `admin`
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_FINISH.md:15]:
```suggestion
- Login — `admin`;
```

File: docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+{::options parse_block_html="false" /}
+
+<div markdown="1">
+## Все установлено, настроено и работает!
+
+Рассмотрим дальнейшие возможности Deckhouse, открывающиеся сразу после установки.
+
+Grafana доступна по адресу [grafana-127-0-0-1.nip.io](http://grafana-127-0-0-1.nip.io).
+
+Доступ к Grafana ограничен basic-аутентификацией (больше вариантов аутентификации можно получить включив модуль [user-auth](/{{ page.lang }}/documentation/v1/modules/150-user-authn/):
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md:14]:
```suggestion
Доступ к Grafana ограничен basic-аутентификацией (больше вариантов аутентификации можно получить, включив модуль [user-auth](/{{ page.lang }}/documentation/v1/modules/150-user-authn/):
```

File: docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+<div markdown="1">
+## Все установлено, настроено и работает!
+
+Рассмотрим дальнейшие возможности Deckhouse, открывающиеся сразу после установки.
+
+Grafana доступна по адресу [grafana-127-0-0-1.nip.io](http://grafana-127-0-0-1.nip.io).
+
+Доступ к Grafana ограничен basic-аутентификацией (больше вариантов аутентификации можно получить включив модуль [user-auth](/{{ page.lang }}/documentation/v1/modules/150-user-authn/):  
+- Логин — `admin`
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md:15]:
```suggestion
- Логин — `admin`;
```

File: docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+</div>
+<div class="cards-blocks__cards">
+
+<div class="cards-item cards-item_inverse">
+<h3 class="cards-item__title text_h3">
+Настройка DexProvider
+</h3>
+<div class="cards-item__text" markdown="1">
+Например, для включения аутентификации через GitHub можно включить модуль [user-authn](/{{ page.lang }}/documentation/v1/modules/150-user-authn/) и [сконфигурировать](/{{ page.lang }}/documentation/v1/modules/150-user-authn/usage.html) объект
+`DexProvider`. После создания `DexProvider`, при попытке доступа ко всем компонентам Deckhouse (Grafana, Dashboard и
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md:128]:
```suggestion
`DexProvider`. После создания `DexProvider` при попытке доступа ко всем компонентам Deckhouse (Grafana, Dashboard и
```

File: docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+т.д.) потребуются аутентификации через GitHub
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_FINISH_RU.md:129]:
```suggestion
т.д.) потребуются аутентификации через GitHub.
```

File: docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
+[kind](https://kind.sigs.k8s.io/) позволяет быстро получить кластер Kubernetes на локальной машине. Такой кластер можно использовать для целей ознакомления, тестирования, разработки.   
+
+Установка Deckhouse на kind, рассматриваемая далее, позволит вам за менее чем 10 минут получить кластер Kubernetes с установленным Deckhouse.
+
+Такой вариант развертывания Deckhouse позволит вам быстро развернуть Deckhouse и познакомиться с основными его возможностями. 
+
+Обратите внимание, что некоторые функции, такие как [управление узлами](/{{ page.lang }}/documentation/v1/modules/040-node-manager/), [управление control-plane](/{{ page.lang }}/documentation/v1/modules/040-control-plane-manager/), работать не будут.
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md:7]:
```suggestion
Обратите внимание, что некоторые функции, такие как [управление узлами](/{{ page.lang }}/documentation/v1/modules/040-node-manager/) и [управление control-plane](/{{ page.lang }}/documentation/v1/modules/040-control-plane-manager/), работать не будут. 
```

File: docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+[kind](https://kind.sigs.k8s.io/) позволяет быстро получить кластер Kubernetes на локальной машине. Такой кластер можно использовать для целей ознакомления, тестирования, разработки.   
+
+Установка Deckhouse на kind, рассматриваемая далее, позволит вам за менее чем 10 минут получить кластер Kubernetes с установленным Deckhouse.
+
+Такой вариант развертывания Deckhouse позволит вам быстро развернуть Deckhouse и познакомиться с основными его возможностями. 
+
+Обратите внимание, что некоторые функции, такие как [управление узлами](/{{ page.lang }}/documentation/v1/modules/040-node-manager/), [управление control-plane](/{{ page.lang }}/documentation/v1/modules/040-control-plane-manager/), работать не будут. 
+
+Данное руководство предлагает установку Deckhouse в **минимальной** конфигурации с включенным [мониторингом](/{{ page.lang }}/documentation/v1/modules/300-prometheus//) на базе Grafana. Для упрощения, при работе с DNS используется сервис [nip.io](https://nip.io).
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md:9]:
```suggestion
Данное руководство предлагает установку Deckhouse в **минимальной** конфигурации с включенным [мониторингом](/{{ page.lang }}/documentation/v1/modules/300-prometheus//) на базе Grafana. Для упрощения при работе с DNS используется сервис [nip.io](https://nip.io).
```

File: docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+
+После выполнения всех шагов данного руководства вы сможете самостоятельно включить интересующие вас модули. Воспользуйтесь [документацией](/{{ page.lang }}/documentation/), чтобы получить об этом необходимую информацию. При возникновении вопросов вы можете попросить помощи [сообщества](/ru/community/about.html).
+
+## Процесс установки
+
+Вам потребуется персональный компьютер с macOS, Windows или Linux достаточной мощности, с доступом в Интернет. На этом компьютере будет развернут кластер Kubernetes, в который будет установлен Deckhouse.
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA_RU.md:15]:
```suggestion
Вам потребуется персональный компьютер достаточной мощности с macOS, Windows или Linux, а также с доступом в Интернет. На этом компьютере будет развернут кластер Kubernetes, в который будет установлен Deckhouse. 
```

File: docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+
+Please note that some features, such as [node management](/{{ page.lang }}/documentation/v1/modules/040-node-manager/) and [control plane management](/{{ page.lang }}/documentation/v1/modules/040-control-plane-manager/) will not work.
+
+This guide covers installing Deckhouse in a **minimal** configuration, with Grafana based [monitoring](/{{ page.lang }}/documentation/v1/modules/300-prometheus//) enabled. To simplify, the [nip.io ](https://nip.io ) service is used for working with DNS.
+
+After completing all the steps in this guide, you will have to enable all the required modules manually. Please, refer to the [documentation](/{{ page.lang }}/documentation/v1/) to learn more or reach out to the Deckhouse [community](/en/community/about.html).
+
+## Installation process
+
+You will need a personal computer with macOS, Windows or Linux of sufficient power, with Internet access. A Kubernetes cluster will be deployed on this computer, and Deckhouse will be installed into a cluster.
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_INSTALL_SCHEMA.md:13]:
```suggestion
You will need a personal computer of sufficient power with macOS, Windows or Linux and with Internet access. A Kubernetes cluster will be deployed on this computer, and Deckhouse will be installed into a cluster. 
```

File: docs/site/_includes/getting_started/kind/STEP_PREPARE.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
+Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), if it is not already installed.:
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_PREPARE.md:1]:
```suggestion
Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), if it is not already installed.
```

File: docs/site/_includes/getting_started/kind/STEP_PREPARE_RU.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    listenAddress: "127.0.0.1"
+    protocol: TCP
+EOF
+```
+{% endsnippetcut %}
+
+> Перед созданием кластера, убедитесь, что у вас не заняты порты 80 и 443.
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/STEP_PREPARE_RU.md:24]:
```suggestion
> Перед созданием кластера убедитесь, что у вас не заняты порты 80 и 443.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 17]
```diff
@@ -0,0 +1,37 @@
+# [<en>] section for bootstrapping the Deckhouse cluster (InitConfiguration)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:1]:
```suggestion
# [<en>] Section for bootstrapping the Deckhouse cluster (InitConfiguration).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+# [<en>] version of the Deckhouse API
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:2]:
```suggestion
# [<en>] Version of the Deckhouse API.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+# [<ru>] секция первичной инициализации кластера Deckhouse (InitConfiguration)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:3]:
```suggestion
# [<ru>] Секция первичной инициализации кластера Deckhouse (InitConfiguration).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 20]
```diff
@@ -0,0 +1,37 @@
...
+# [<ru>] используемая версия API Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:4]:
```suggestion
# [<ru>] Используемая версия API Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+apiVersion: deckhouse.io/v1
+# [<en>] type of the configuration section
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:6]:
```suggestion
# [<en>] Type of the configuration section.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 22]
```diff
@@ -0,0 +1,37 @@
...
+# [<ru>] тип секции конфигурации
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:7]:
```suggestion
# [<ru>] Тип секции конфигурации.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 23]
```diff
@@ -0,0 +1,37 @@
...
+kind: InitConfiguration
+# [<en>] Deckhouse parameters
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:9]:
```suggestion
# [<en>] Deckhouse parameters.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+# [<ru>] секция с параметрами Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:10]:
```suggestion
# [<ru>] Секция с параметрами Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 25]
```diff
@@ -0,0 +1,37 @@
...
+deckhouse:
+  # [<en>] the release channel in use
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:12]:
```suggestion
  # [<en>] The release channel in use.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+  # [<ru>] используемый канал обновлений
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:13]:
```suggestion
  # [<ru>] Используемый канал обновлений.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+  releaseChannel: Stable
+  # [<en>] the Minimal bundle is used when installing Deckhouse in an existing cluster
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:15]:
```suggestion
  # [<en>] The Minimal bundle is used when installing Deckhouse in an existing cluster.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+  # [<ru>] при установке Deckhouse в существующий кластер используется вариант поставки — Minimal
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:16]:
```suggestion
  # [<ru>] При установке Deckhouse в существующий кластер используется вариант поставки — Minimal.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc [hunk 29]
```diff
@@ -0,0 +1,37 @@
...
+  bundle: Minimal
+  configOverrides:
+    global:
+      modules:
+        # [<en>] Template for Ingress resources of Deckhouse modules.
+        # [<en>] E.g., Grafana for %s.example.com will be available as 'grafana.example.com'.
+        # [<en>] The nip.io service is used as as a working example.
+        # [<ru>] Шаблон для создания Ingress-ресурсов модулей Deckhouse.
+        # [<ru>] Например, Grafana для %s.example.com будет доступна на домене grafana.example.com
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ce.inc:25]:
```suggestion
        # [<ru>] Например, Grafana для %s.example.com будет доступна на домене grafana.example.com.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 30]
```diff
@@ -0,0 +1,43 @@
+# [<en>] section for bootstrapping the Deckhouse cluster (InitConfiguration)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:1]:
```suggestion
# [<en>] Section for bootstrapping the Deckhouse cluster (InitConfiguration).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 31]
```diff
@@ -0,0 +1,43 @@
...
+# [<en>] version of the Deckhouse API
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:2]:
```suggestion
# [<en>] Version of the Deckhouse API.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 32]
```diff
@@ -0,0 +1,43 @@
...
+# [<ru>] секция первичной инициализации кластера Deckhouse (InitConfiguration)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:3]:
```suggestion
# [<ru>] Секция первичной инициализации кластера Deckhouse (InitConfiguration).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 33]
```diff
@@ -0,0 +1,43 @@
...
+# [<ru>] используемая версия API Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:4]:
```suggestion
# [<ru>] Используемая версия API Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 34]
```diff
@@ -0,0 +1,43 @@
...
+apiVersion: deckhouse.io/v1
+# [<en>] type of the configuration section
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:6]:
```suggestion
# [<en>] Type of the configuration section.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 35]
```diff
@@ -0,0 +1,43 @@
...
+# [<ru>] тип секции конфигурации
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:7]:
```suggestion
# [<ru>] Тип секции конфигурации.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 36]
```diff
@@ -0,0 +1,43 @@
...
+kind: InitConfiguration
+# [<en>] Deckhouse parameters
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:9]:
```suggestion
# [<en>] Deckhouse parameters.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 37]
```diff
@@ -0,0 +1,43 @@
...
+# [<ru>] секция с параметрами Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:10]:
```suggestion
# [<ru>] Секция с параметрами Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 38]
```diff
@@ -0,0 +1,43 @@
...
+deckhouse:
+  # [<en>] address of the Docker registry where the Deckhouse images are located
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:12]:
```suggestion
  # [<en>] Address of the Docker registry where the Deckhouse images are located.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 39]
```diff
@@ -0,0 +1,43 @@
...
+  # [<ru>] адрес Docker registry с образами Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:13]:
```suggestion
  # [<ru>] Адрес Docker registry с образами Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 40]
```diff
@@ -0,0 +1,43 @@
...
+  imagesRepo: registry.deckhouse.io/deckhouse/ee
+  # [<en>] a special string with your token to access Docker registry (generated automatically for your license token)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:15]:
```suggestion
  # [<en>] A special string with your token to access Docker registry (generated automatically for your license token).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 41]
```diff
@@ -0,0 +1,43 @@
...
+  # [<ru>] строка с ключом для доступа к Docker registry (сгенерировано автоматически для вашего токена доступа)
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:16]:
```suggestion
  # [<ru>] Строка с ключом для доступа к Docker registry (сгенерировано автоматически для вашего токена доступа).
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 42]
```diff
@@ -0,0 +1,43 @@
...
+  registryDockerCfg: <YOUR_ACCESS_STRING_IS_HERE>
+  # [<en>] the release channel in use
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:18]:
```suggestion
  # [<en>] The release channel in use.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 43]
```diff
@@ -0,0 +1,43 @@
...
+  # [<ru>] используемый канал обновлений
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:19]:
```suggestion
  # [<ru>] Используемый канал обновлений.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 44]
```diff
@@ -0,0 +1,43 @@
...
+  releaseChannel: Stable
+  # [<en>] the Minimal bundle is used when installing Deckhouse in an existing cluster
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:21]:
```suggestion
  # [<en>] The Minimal bundle is used when installing Deckhouse in an existing cluster.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 45]
```diff
@@ -0,0 +1,43 @@
...
+  # [<ru>] при установке Deckhouse в существующий кластер используется вариант поставки — Minimal
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:22]:
```suggestion
  # [<ru>] При установке Deckhouse в существующий кластер используется вариант поставки — Minimal.
```

File: docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc [hunk 46]
```diff
@@ -0,0 +1,43 @@
...
+  bundle: Minimal
+  configOverrides:
+    global:
+      modules:
+        # [<en>] Template for Ingress resources of Deckhouse modules.
+        # [<en>] E.g., Grafana for %s.example.com will be available as 'grafana.example.com'.
+        # [<en>] The nip.io service is used as as a working example.
+        # [<ru>] Шаблон для создания Ingress-ресурсов модулей Deckhouse.
+        # [<ru>] Например, Grafana для %s.example.com будет доступна на домене grafana.example.com
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/config.yml.default.ee.inc:31]:
```suggestion
        # [<ru>] Например, Grafana для %s.example.com будет доступна на домене grafana.example.com.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 47]
```diff
@@ -0,0 +1,18 @@
+# [<en>] section containing the parameters of nginx ingress controller
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:1]:
```suggestion
# [<en>] Section containing the parameters of nginx ingress controller.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 48]
```diff
@@ -0,0 +1,18 @@
...
+# [<en>] version of the Deckhouse API
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:2]:
```suggestion
# [<en>] Version of the Deckhouse API.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 49]
```diff
@@ -0,0 +1,18 @@
...
+# [<ru>] секция, описывающая параметры nginx ingress controller
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:3]:
```suggestion
# [<ru>] Секция, описывающая параметры nginx ingress controller.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 50]
```diff
@@ -0,0 +1,18 @@
...
+# [<ru>] используемая версия API Deckhouse
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:4]:
```suggestion
# [<ru>] Используемая версия API Deckhouse.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 51]
```diff
@@ -0,0 +1,18 @@
...
+apiVersion: deckhouse.io/v1
+kind: IngressNginxController
+metadata:
+  name: nginx
+spec:
+  # [<en>] the name of the Ingress class to use with the Ingress nginx controller
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:10]:
```suggestion
  # [<en>] The name of the Ingress class to use with the Ingress nginx controller.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 52]
```diff
@@ -0,0 +1,18 @@
...
+  # [<ru>] имя Ingress-класса для обслуживания Ingress NGINX controller
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:11]:
```suggestion
  # [<ru>] Имя Ingress-класса для обслуживания Ingress NGINX controller.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 53]
```diff
@@ -0,0 +1,18 @@
...
+  ingressClass: nginx
+  # [<en>] the way traffic goes to cluster from the outer network
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:13]:
```suggestion
  # [<en>] The way traffic goes to cluster from the outer network.
```

File: docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc [hunk 54]
```diff
@@ -0,0 +1,18 @@
...
+  # [<ru>] способ поступления трафика из внешнего мира
```
Comment (Zhbert) [inline docs/site/_includes/getting_started/kind/partials/ingress-nginx-controller.yml.inc:14]:
```suggestion
  # [<ru>] Способ поступления трафика из внешнего мира.
```

**Review by Zhbert — CHANGES_REQUESTED** [review]
@z9r5 Please check new updates.

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

**Review by Zhbert — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/stage"...

**Review by konstantin-axenov — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Running workflow "deploy/web/test"...

**Pull request merged**